src/test/java/org/sdo/rendezvous/testng/custom/logs/CustomLogs.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Intel Corporation
+// SPDX-License-Identifier: Apache 2.0
+
+package org.sdo.rendezvous.testng.custom.logs;
+
+import org.testng.ITestResult;
+import org.testng.TestListenerAdapter;
+
+public class CustomLogs extends TestListenerAdapter {
+
+  private String testClassName = null;
+  private String testMethodName = null;
+  private long executionTime = 0;
+
+  @Override
+  public void onTestFailure(ITestResult tr) {
+
+    testClassName = tr.getTestClass().toString();
+    testMethodName = tr.getMethod().getMethodName().toString();
+    executionTime = tr.getEndMillis() - tr.getStartMillis();
+
+    System.out.println(testClassName + " [TestMethod name=" + testMethodName + "] [" + executionTime
```

Comment:
Use log4j functionality instead of System.out.println.

Reply:
Done